lesson_06_hw.html

```diff
@@ -0,0 +1,1402 @@
+                const $liItem = document.createElement('li');
+                $liItem.innerHTML = liText;
+                $itemList.appendChild($liItem);
+              }
+
+              const $linksArr = $itemList.getElementsByTagName('a');
+              for (var i = 0; i < $linksArr.length; i++) {
+                $linksArr[i].setAttribute('id', 'pictureAnchor_' + [i]);
+                $linksArr[i].addEventListener('click', this.openModalWindow);
+              }
+            }
+          },
+
+          changeQuantity(event) {
+            event.preventDefault();
+            const selectValue = document.getElementById('changeQuantitySelect').value;
+            const inputValue = +document.getElementById('changeQuantityInput').value;
+
+            for (var i = 0; i < basket.items.length; i++) {
+              if (basket.items[i].name == selectValue) {
+                if (inputValue == 0) {
+                  basket.items.splice(i, 1);
+                } else {
+                  basket.items[i].quantity = inputValue;
+                }
+              }
+            }
+
+            basket.displayItems();
+            basket.redrawSelects();
+          },
+
+          handleSelectChange(event) {
+            const selectValue = document.getElementById('changeQuantitySelect').value;
+            const $changeQuantitySpan = document.getElementById('changeQuantitySpan');
+            for (var i = 0; i < basket.items.length; i++) {
+              if (basket.items[i].name == selectValue) {
+                $changeQuantitySpan.innerHTML = `Quantity: ${basket.items[i].quantity}`;
+              }
+            }
+          },
+
+          redrawSelects() {
+            const $changeQuantitySelect = document.getElementById('changeQuantitySelect');
+            $changeQuantitySelect.innerHTML = '';
+            for (var i = 0; i < this.items.length; i++) {
+              const $changeQuantityOption = document.createElement('option');
+              $changeQuantityOption.innerHTML = this.items[i].name;
+              $changeQuantitySelect.appendChild($changeQuantityOption);
+            }
+
+            const $removeItemSelect = document.getElementById('removeItemSelect');
+            $removeItemSelect.innerHTML = '';
+            for (var i = 0; i < this.items.length; i++) {
+              const $removeItemOption = document.createElement('option');
+              $removeItemOption.innerHTML = this.items[i].name;
+              $removeItemSelect.appendChild($removeItemOption);
+            }
+          },
+
+          displayControls() {
+            const $basketControls = document.getElementById('basketControls');
+
+            // Добавляем кнопку "очистить корзину"
+            const $clearBasketButton = document.createElement('button');
```

Comment:
I'd rather recommend you should use templates here as it was shown on a previous lesson